src/eponai/tools/cursive_arcadia_repl.clj

```diff
@@ -52,6 +60,18 @@
               (assoc :id id))))))))
 
 
+(defn arcadia-nrepl-eval [handler msg]
```

Comment:
How about changing the names of these arcadia functions?
* `arcadia-handle-message` -> `arcadia-eval`
* `arcadia-nrepl-eval` -> `try-arcadia-eval`